go.mod

```diff
@@ -210,3 +210,5 @@ replace (
 	github.com/portworx/sched-ops => github.com/portworx/sched-ops v1.20.4-rc1
 	k8s.io/client-go => k8s.io/client-go v0.29.4
 )
+
+replace github.com/openshift/assisted-service/models => github.com/openshift/assisted-service/models v0.0.0-20230831114549-1922eda29cf8
```

Comment:
could you remove this please and run `go mod tidy && go mod vendor` again`?

Reply:
Okay, done.